'Solution'

yura-push opened the pull request.

yura-push committed:
'Solution'

viktoria-rybenchuk commented on app/customer.py:
```diff
@@ -0,0 +1,65 @@
+        for product, quantity in self.product_cart.items():
+            cost_of_product = quantity * shops_products[product]
+            total_cost += cost_of_product
+        return total_cost
```
write this code with sum and generator expression

viktoria-rybenchuk commented on app/show_information.py:
```diff
@@ -0,0 +1,35 @@
+            for product, price in shop.products.items():
+                product_price = customer.product_cart[product] * price
+                if product_price == 3.0:
+                    product_price = 3
```
handle this problem for all cases not just for 3

yura-push committed:
-added logic to handle all cases not just 3
-reworked customer.cost_of_all_products_to_buy - used generator expression and sum

Dimosphen1 reviewed (approved):
LGTM, well done!
